perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -775,6 +773,20 @@ def Prepare(self, vm):
                                                      DEFAULT_GCS_REGION,
                                                      self.regional_bucket_name))
 
+    # Detect if we need to install crcmod for gcp.
+    # See "gsutil help crc" for details.
+    raw_result, _ = vm.RemoteCommand('%s version -l' % vm.gsutil_path)
+    logging.info('gsutil version -l raw result is %s', raw_result)
+    search_string = 'compiled crcmod: True'
+    result_string = re.findall(search_string, raw_result)
+    if len(result_string) == 0:
```

Comment:
Done. I've made a package called crcmod under the packages directory and am calling vm.install(crcmod) here.